AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngineBehavior.swift

```diff
@@ -17,6 +17,7 @@ enum RemoteSyncEngineEvent {
     case subscriptionsActivated
     case mutationQueueStarted
     case syncStarted
+    case cleanedUp
```

Comment:
BTW, while I'm thinking about it, I like the idea you're promoting (intentionally or not) of normalizing our naming: `State` can represent a point-in-time view of a system, and could be thought of as something managed internally. `Event` can represent an external report of a state transition, even if it doesn't literally represent a change from one `State` to another.